config-local.php-dist

```diff
@@ -6,11 +6,13 @@
  */
 
 // LDAP
-define('LDAP_HOST', 'pm-ns.mozilla.org');
+define('LDAP_HOST', 'DEFAULT_LDAP_HOST');
+define('LDAP_EXCLUDE', '');  // '(!(defaultAttribute=defaultValue))'
 
 // Memcache (port number is mandatory)
 define("MEMCACHE_ENABLED", true);
 $memcache_servers = array(
-    'localhost:11211',
+    'DEFAULT_MEMCACHE_SERVER_ONE:11211',
```

Comment:
I was trying to indicate that this array should be set to a meaningful list of one or more memcache servers, rather than simply defaulting to localhost:11211. But then in the code we ALSO set a default that overrides a blank array in the config. So the app, no matter what, will always assume that memcache is available.

I'm happy to skip this patch since it's irrelevant to the Mozilla IT side of things.